eng/helix/content/runtests.ps1

```diff
@@ -79,6 +79,13 @@ if ([string]::IsNullOrEmpty($FeedCred)) {
     InstallDotnetSDKAndRuntime "https://dotnetclimsrc.blob.core.windows.net/dotnet" $FeedCred
 }
 
+Write-Host "Enable-WindowsOptionalFeature -Online -FeatureName ServerMediaFoundation  (For Playwright)"
+try {
+    Enable-WindowsOptionalFeature -Online -FeatureName ServerMediaFoundation 
```

Comment:
@MattGal we need a new windows feature for one of our new browser testing automation frameworks to work (it failed with an exception saying to turn this on), is this a proper way to enable things?  Or is this something we should ask for at the helix queue level?

Reply:
Reading up on the feature, if it works this is probably fine for now.  In the long term you should file an issue asking for this to be added to one or more Helix queues on dotnet/core-eng because even though it looks very much like it wouldn't have any impact on other workitem's execution, installing it {number of machines created} times versus once as an artifact when preparing disk images is a tangible monetary impact.

Reply:
Thanks @MattGal , I'll file that issue once we are sure this is actually working :)

Reply:
I must have broken something in our boostrapper with this change, our runpowershell cmds don't seem to work anymore, 

@MattGal was there still a way I can get to the other logs/payloads again from just the job id?
https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-pull-30113-merge-75bc1dcdb5e843ed99/BlazorTemplates.Tests--net6.0/console.99bde411.log?sv=2019-07-07&se=2021-03-04T18%3A09%3A40Z&sr=c&sp=rl&sig=dV%2BEVT%2Bk72Ot3qQx%2BY678%2F5BTANFqCjLcrhiufd%2BAIw%3D

Reply:
https://helix.dot.net/api/jobs/75bc1dcd-b5e8-43ed-992d-341074d9d24b/details?api-version=2019-06-17 gives us [the job list](https://helixde8s23ayyeko0k025g8.blob.core.windows.net/helix-job-fe8506f2-18b4-40b9-a30d-1f20d1acae2389d1092fc534239bd/job-list-bd3cd190-bccb-4df0-889a-08eb05be0262.json?sv=2019-07-07&se=2021-03-04T18%3A08%3A28Z&sr=c&sp=rl&sig=0LNozIRpHgzOzbTTKHcX4qDi7Iifv2jMqHjZfsI6Hgc%3D) but the work items are missing in Kusto and noone is freaking out so something very strange is happening.  I will look after I eat a quick lunch

Reply:
OK there's an interesting Kusto migration problem going on that I'm investigating separately, but I think you'd want something like this in Kusto (or from the Helix API)

[Given this log](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-pull-30113-merge-75bc1dcdb5e843ed99/BlazorTemplates.Tests--net6.0/0709e05d-92ab-4993-83f1-8268b8a59a0d.log?sv=2019-07-07&se=2021-03-04T18%3A09%3A40Z&sr=c&sp=rl&sig=dV%2BEVT%2Bk72Ot3qQx%2BY678%2F5BTANFqCjLcrhiufd%2BAIw%3D) it looks like RunPowershell.cmd just wasn't in the work item's payload.

Reply:
Ah yeah, that makes sense, I guess I messed up the msbuild logic so it didn't include any of the helix boostrapping, getting closer now thanks!